src/site/markdown/index.md.vm

```diff
@@ -59,4 +60,6 @@ Getting Started
 mvn package
 ```
 
+See the "Goals" page for more details about the supported configuration parameters.
+
```

Comment:
My bad, sad that I didn't find those! Would you mind adding something like this to help people find the list of parameters? With a link it would be even better of course…

Reply:
This isn't necessary. The name and path to the introduction page and the goals page are "well-known" by maven plugin documentation convention and are the only two items in our navigation menu that aren't boilerplate maven artifact project info. See other standard plugins for examples of this convention:

* https://maven.apache.org/plugins/maven-clean-plugin/
* https://maven.apache.org/plugins/maven-compiler-plugin/
* etc.

I'd don't think we should put anything additional here, because if the convention changes or evolves, that's more work for us to maintain, and I just don't see it being a problem... you can find it easily by the second link in the navigation menu. It really doesn't need any additional verbiage. Granted, some users may not be deeply familiar with these conventions, and may take a minute to find it the first time, or may have to ask... but that's just more opportunity to engage novice users/devs. Being too redundant in the docs just to avoid that engagement doesn't seem like a good idea to me, especially if it means more places we have to update when the convention evolves, or more bloat to the page.